WcaOnRails/app/views/teams/_team_form.html.erb

```diff
@@ -0,0 +1,15 @@
+  <%= simple_form_for [@committee, @team], html: { class: 'form-horizontal are-you-sure no-submit-on-enter' }, wrapper: :horizontal_form,
+    wrapper_mappings: {
+      check_boxes: :horizontal_radio_and_checkboxes,
+      radio_buttons: :horizontal_radio_and_checkboxes,
+      file: :horizontal_file_input,
+      boolean: :horizontal_boolean
+    }  do |f| %>
+    <%= render 'shared/error_messages', object: f.object %>
+
+    <%= f.input :name %>
+    <%= f.input :description %>
```

Comment:
What about using the markdown editor here? (If you will do so then remember to render it as markdown as well).